core/cmd/shell_local.go

```diff
@@ -396,6 +384,9 @@ func (s *Shell) runNode(c *cli.Context) error {
 			return fmt.Errorf("error listing legacy evm chains: %w", err)
 		}
 		for _, ch := range chainList {
+			if err = keyStore.Migrate(s.Config.Password().VRF(), ch.ID()); err != nil {
+				return errors.Wrap(err, "error migrating keystore")
+			}
 			if ch.Config().EVM().AutoCreateKey() {
```

Comment:
This makes sense to me, but I'm not 100% sure

Reply:
can you find the author of this code and confirm the removal? i'm surprised that we have to touch keystore migrations with this change, and we have to 100% certain that this is ok

Reply:
Looks like @samsondav and @RyanRHall touched this code the most. Samson already approved, maybe Ryan can give some input here?

Reply:
am i reading it correctly - the existing behavior was to migrate to the default chain. now we are going to migrate to all of the chains? 

my reading of Migrate is that it does a bunch of stuff to a bunch of different kind of keys. i'm not convinced it was intended to be called multiple times.

maybe we need to split it up and call the evm-specific part once per chain? or change it to take a slice of evm keys

Reply:
After a brief discussion with @RyanRHall it seems like this code is obsolete, so it can be just deleted.
We don't permit creating new v1 keys anymore. So all v1 keys should have been migrated by now.